third_party/js.bzl

```diff
@@ -17,500 +17,509 @@ load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+            "https://registry.npmjs.org/sinon-chai/-/sinon-chai-2.8.0.tgz",
+        ],
+        strip_prefix = "package",
+        path = "/sinon-chai",
+    )
+
+    web_library_external(
+        name = "org_npmjs_registry_stacky",
+        licenses = ["notice"],  # BSD-3-Clause
+        sha256 = "c659e60f7957d9d80c23a7aacc4d71b19c6421a08f91174c0062de369595acae",
+        urls = [
+            "http://mirror.tensorflow.org/registry.npmjs.org/stacky/-/stacky-1.3.1.tgz",
+            "https://registry.npmjs.org/stacky/-/stacky-1.3.1.tgz",
+        ],
+        strip_prefix = "package",
+        path = "/stacky",
+    )
+
+    web_library_external(
+        name = "org_npmjs_registry_web_component_tester",
+        licenses = ["notice"],  # BSD-3-Clause
+        sha256 = "9d4ebd4945df8a936916d4d32b7f280f2a3afa35f79e7ca8ad3ed0a42770c537",
+        urls = [
+            "http://mirror.tensorflow.org/registry.npmjs.org/web-component-tester/-/web-component-tester-4.3.6.tgz",
+            "https://registry.npmjs.org/web-component-tester/-/web-component-tester-4.3.6.tgz",
+        ],
+        strip_prefix = "package",
+        path = "/web-component-tester",
+        suppress = [
+            "absolutePaths",
+            "strictDependencies",
+        ],
+        deps = [
+            "@com_lodash",
+            "@org_npmjs_registry_accessibility_developer_tools",
+            "@org_npmjs_registry_async",
+            "@org_npmjs_registry_chai",
+            "@org_npmjs_registry_mocha",
+            "@org_npmjs_registry_sinon",
+            "@org_npmjs_registry_sinon_chai",
+            "@org_npmjs_registry_stacky",
+            "@org_polymer_test_fixture",
+        ],
+    )
+
+    web_library_external(
+        name = "org_polymer_test_fixture",
+        licenses = ["notice"],  # BSD-3-Clause
+        sha256 = "59d6cfb1187733b71275becfea181fe0aa1f734df5ff77f5850c806bbbf9a0d9",
+        strip_prefix = "test-fixture-2.0.1",
+        urls = [
+            "http://mirror.tensorflow.org/github.com/PolymerElements/test-fixture/archive/v2.0.1.tar.gz",
+            "https://github.com/PolymerElements/test-fixture/archive/v2.0.1.tar.gz",
+        ],
+        path = "/test-fixture",
+        exclude = ["test/**"],
+    )
+
+    filegroup_external(
+        name = "org_tensorflow_tfjs",
+        licenses = ["notice"],  # Apache 2.0
+        sha256_urls = {
+            "3870fc1d4eaefce7370b5bce3992e4dadcad8e3a5999b034302e38b2094873ea": [
+                "https://cdn.jsdelivr.net/npm/@tensorflow/tfjs@1.0.1/dist/tf.min.js",
+            ],
+        },
+    )
+
+    filegroup_external(
+        name = "org_tensorflow_tfjs_mobilenet",
+        licenses = ["notice"],  # Apache 2.0
+        sha256_urls = {
+            "aca32ec8a778d17acb623bf495c6e60ec38312b761b9a8c88af4c841c25eba70": [
+                "https://cdn.jsdelivr.net/npm/@tensorflow-models/mobilenet@1.0.0/dist/mobilenet.js",
```

Comment:
Please use the tensorflow.org mirror in addition to this.

Reply:
do you know what that mirror url would be? This is an npm package here https://www.npmjs.com/package/@tensorflow-models/mobilenet/v/1.0.0

Reply:
done.